Add slogan (#9)
galette/website: content/en/_index.md

title: "Home"
description: "Free Software for association management, Galette helps you manage members, contributions, and other aspects of your association from your web browser. Simple. Open-source. Free."
---

> Galette is a OpenSource web application for membership and contribution management, specially designed for non-profit organizations.<br/>

Reviewer: ```suggestion
Galette is an open source web application for membership and contribution management, specially designed for non-profit organizations.<br/>
```
I only saw this. 
I think "open source" should be written as two words and in lowercase (see on [wikipedia](https://fr.wikipedia.org/wiki/Open_Source_Initiative#%C2%AB_Open_source_%C2%BB_et_%C2%AB_logiciel_libre_%C2%BB)); same in french.
Otherwise everything seems to be OK 😁
